tests/CMakeLists.txt

```diff
@@ -148,13 +171,18 @@ ConfigureTest(DEVICE_SCALAR_TEST device_scalar_tests.cpp)
 ConfigureTest(LOGGER_TEST logger_tests.cpp)
 
 # uvector tests
-ConfigureTest(DEVICE_UVECTOR_TEST device_uvector_tests.cpp)
+ConfigureTest(DEVICE_UVECTOR_TEST device_uvector_tests.cpp GPUS 1 PERCENT 60)
 
 # arena MR tests
-ConfigureTest(ARENA_MR_TEST mr/device/arena_mr_tests.cpp)
+ConfigureTest(ARENA_MR_TEST mr/device/arena_mr_tests.cpp GPUS 1 PERCENT 60)
 
 # binning MR tests
 ConfigureTest(BINNING_MR_TEST mr/device/binning_mr_tests.cpp)
 
 # callback memory resource tests
 ConfigureTest(CALLBACK_MR_TEST mr/device/callback_mr_tests.cpp)
+
+rapids_test_install_relocatable(
+  INSTALL_COMPONENT_SET testing
+  DESTINATION bin/gtests/librmm
+  EXCLUDE_FROM_ALL)
```

Comment:
Under what circumstances would we actually want tests installed? Are we going to end up specifying `EXCLUDE_FROM_ALL` almost always?

Reply:
You are correct that we should make `EXCLUDE_FROM_ALL` the default as the only time we want to install tests is for the creation of the `librmm-tests` conda package which is done to allow for different build and test CI jobs.

Reply:
Awesome thanks. I have a question on whether we should make a breaking change in the rapids-cmake PR, let's resolve that before merging this in case you agree with making a breaking change.

Reply:
I just merged https://github.com/rapidsai/rapids-cmake/pull/388 so once this PR is updated to remove the `EXCLUDE_FROM_ALL` it should be ready to merge.

Reply:
Made the change.